pmd-visualforce/etc/grammar/VfParser.jjt

```diff
@@ -255,8 +260,9 @@ PARSER_END(VfParser)
 
 <HtmlScriptContentState> TOKEN :
 {
-		 <HTML_SCRIPT_CONTENT: (~[]) >
-		| <HTML_SCRIPT_END_TAG : "</script>" > : AfterTagState
+		  <HTML_SCRIPT_END_TAG : "</script>" > : AfterTagState
+		| <EL_EXPRESSION_IN_SCRIPT: "{!" (<WHITESPACES>)? > : ElInScriptState
```

Comment:
@sgorbaty awesome, that is what I was after! Being that way, then this grammar is correct 100% of the time identifying those as EL. This is the best scenario possible.